flink-connector-tidb-cdc/src/main/java/com/ververica/cdc/connectors/tidb/TDBSourceOptions.java

```diff
@@ -0,0 +1,142 @@
+
+    public static final ConfigOption<Long> TIKV_GRPC_TIMEOUT =
+            ConfigOptions.key(ConfigUtils.TIKV_GRPC_TIMEOUT)
+                    .longType()
+                    .noDefaultValue()
+                    .withDescription("TiKV GRPC timeout in ms");
+
+    public static final ConfigOption<Long> TIKV_GRPC_SCAN_TIMEOUT =
+            ConfigOptions.key(ConfigUtils.TIKV_GRPC_SCAN_TIMEOUT)
+                    .longType()
+                    .noDefaultValue()
+                    .withDescription("TiKV GRPC scan timeout in ms");
+
+    public static final ConfigOption<Integer> TIKV_BATCH_GET_CONCURRENCY =
+            ConfigOptions.key(ConfigUtils.TIKV_BATCH_GET_CONCURRENCY)
+                    .intType()
+                    .noDefaultValue()
+                    .withDescription("TiKV GRPC batch get concurrency");
+
+    public static final ConfigOption<Integer> TIKV_BATCH_PUT_CONCURRENCY =
+            ConfigOptions.key(ConfigUtils.TIKV_BATCH_PUT_CONCURRENCY)
+                    .intType()
+                    .noDefaultValue()
+                    .withDescription("TiKV GRPC batch put concurrency");
+
+    public static final ConfigOption<Integer> TIKV_BATCH_SCAN_CONCURRENCY =
+            ConfigOptions.key(ConfigUtils.TIKV_BATCH_SCAN_CONCURRENCY)
+                    .intType()
+                    .noDefaultValue()
+                    .withDescription("TiKV GRPC batch scan concurrency");
+
+    public static final ConfigOption<Integer> TIKV_BATCH_DELETE_CONCURRENCY =
+            ConfigOptions.key(ConfigUtils.TIKV_BATCH_DELETE_CONCURRENCY)
+                    .intType()
+                    .noDefaultValue()
+                    .withDescription("TiKV GRPC batch delete concurrency");
+
+    public static TiConfiguration getTiConfiguration(final Map<String, String> options) {
+        final Configuration configuration = Configuration.fromMap(options);
+
+        final TiConfiguration tiConf =
+                configuration
+                        .getOptional(TIKV_PD_ADDRESSES)
+                        .map(TiConfiguration::createDefault)
+                        .orElseGet(TiConfiguration::createDefault);
+
+        configuration.getOptional(TIKV_GRPC_TIMEOUT).ifPresent(tiConf::setTimeout);
+        configuration.getOptional(TIKV_GRPC_SCAN_TIMEOUT).ifPresent(tiConf::setScanTimeout);
+        configuration
+                .getOptional(TIKV_BATCH_GET_CONCURRENCY)
+                .ifPresent(tiConf::setBatchGetConcurrency);
+        configuration
+                .getOptional(TIKV_BATCH_PUT_CONCURRENCY)
+                .ifPresent(tiConf::setBatchPutConcurrency);
+        configuration
+                .getOptional(TIKV_BATCH_SCAN_CONCURRENCY)
+                .ifPresent(tiConf::setBatchScanConcurrency);
+        configuration
+                .getOptional(TIKV_BATCH_DELETE_CONCURRENCY)
+                .ifPresent(tiConf::setBatchDeleteConcurrency);
+        return tiConf;
+    }
```

Comment:
We cannot list all valida TiKV parameters here, I think we should  enable user to pass `tikv.xx` options just like Flink Kafka Connector enable user to `properties.xx`.